x/qgb/abci_test.go

```diff
@@ -83,22 +82,20 @@ func TestValsetCreationWhenEditingEVMAddr(t *testing.T) {
 	ctx = ctx.WithBlockHeight(ctx.BlockHeight() + 1)
 	msgServer := stakingkeeper.NewMsgServerImpl(input.StakingKeeper)
 
-	newEVMAddr, err := teststaking.RandomEVMAddress()
-	require.NoError(t, err)
 	editMsg := stakingtypes.NewMsgEditValidator(
 		testutil.ValAddrs[1],
 		stakingtypes.Description{},
 		nil,
 		nil,
-		newEVMAddr,
 	)
-	_, err = msgServer.EditValidator(ctx, editMsg)
+	_, err := msgServer.EditValidator(ctx, editMsg)
 	require.NoError(t, err)
 	staking.EndBlocker(ctx, input.StakingKeeper)
 	qgb.EndBlocker(ctx, *pk)
 	ctx = ctx.WithBlockHeight(ctx.BlockHeight() + 10)
 
-	assert.Equal(t, currentAttestationNonce+1, pk.GetLatestAttestationNonce(ctx))
+	// FIXME: this needs to change to 2 once we have a proper implementation of editing the EVM address
```

Comment:
[question] not sure what you mean by "2"

Reply:
Ah it's currently 1 but we should be asserting that the latest nonce is 2

Reply:
I also should have @-ed myself but forgot to